adam-core/src/main/scala/org/bdgenomics/adam/converters/SupportedHeaderLines.scala

```diff
@@ -63,10 +108,72 @@ private[adam] object SupportedHeaderLines {
     transcriptEffects
   )
 
+  lazy val genotype = VCFStandardHeaderLines.getFormatLine(
+    VCFConstants.GENOTYPE_KEY)
```

Comment:
I think I'm OK trusting the GT format line, but if you feel strongly enough about it, I'll rewrite that one too. ;)